assets/js/modules/reader-revenue-manager/datastore/publications.js

```diff
@@ -17,22 +17,26 @@
  */
 
 /**
- * Internal dependencies
+ * Internal dependencies.
  */
 import API from 'googlesitekit-api';
+import { CORE_MODULES } from '../../../googlesitekit/modules/datastore/constants';
+import { CORE_UI } from '../../../googlesitekit/datastore/ui/constants';
 import { commonActions, combineStores } from 'googlesitekit-data';
 import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
 import {
 	MODULES_READER_REVENUE_MANAGER,
+	MODULE_SLUG,
 	PUBLICATION_ONBOARDING_STATES,
+	UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION,
 } from './constants';
 
 const fetchGetPublicationsStore = createFetchStore( {
 	baseName: 'getPublications',
 	controlCallback: () =>
 		API.get(
 			'modules',
-			'reader-revenue-manager',
+			MODULE_SLUG,
 			'publications',
 			{},
 			{ useCache: true }
```

Comment:
Unrelated observation – but selectors like this one which are only really used during setup should not use cache so that it pulls fresh data every time, otherwise there is no way to pull a new publication after creating one on the service once cached.

Reply:
Good spot @aaemnnosttv.

It would be nice to roll this one-line fix into a forthcoming RRM PR rather than create a separate issue for it.

@ankitrox, could I ask you to please include it in your next PR?

```suggestion
			{ useCache: false }
```

Reply:
Thank you folks! Just to make sure we don't miss this out, I've included this in [my PR](https://github.com/google/site-kit-wp/pull/9098) for #8800.